cypher-shell/src/main/java/org/neo4j/shell/cli/Format.java

```diff
@@ -19,7 +19,7 @@ public static Format parse(@Nonnull String format) {
         } else if (format.equalsIgnoreCase( VERBOSE.name() )) {
             return VERBOSE;
         } else {
-            return isOutputInteractive() ? VERBOSE : PLAIN;
+            return isInputInteractive() ? VERBOSE : PLAIN;
```

Comment:
So now it checks input instead of output. Didn't we decide to check both? E.g.

`return (isInputInteractive() && isOutputInteractive) ? VERBOSE : PLAIN;`

?